tests/unit/crypto_symmetric_test.c

```diff
@@ -110,5 +110,7 @@ int main()
         unit_test(test_plain_text_size_max),
     };
 
+    CryptoDeInitialize();
```

Comment:
Right, maybe re-do the commits so that all the additions that fix a memory leak are in this commit and not in the whitespace commit previous.